warehouse/templates/manage/roles.html

```diff
@@ -17,6 +17,15 @@
 
 {% block title %}{% trans project_name=project.name %}Manage '{{ project_name }}' collaborators{% endtrans %}{% endblock %}
 
+{% macro two_factor_badge(user) -%}
+  {% if user.has_two_factor %}
+    <span class="badge badge--success">
+      <i class="fa fa-check" aria-hidden="true"></i>
+      {% trans %}2FA enabled{% endtrans %}
```

Comment:
```suggestion
      {% trans %}2FA{% endtrans %}
```
What do you think about adding a red badge for users without 2FA enabled?

Reply:
To quote myself from above,
> I choose not to display a badge if no 2FA rather than displaying a red badge, but this can be discussed.

:D

I'm perfectly fine doing it, I just couldn't decide by myself if it was akin to "blaming users who haven't activated 2FA" which there might be good reasons for (e.g. you don't have the necessary hardware)

Reply:
<img width="823" alt="Capture d’écran 2021-04-23 à 21 54 46" src="https://user-images.githubusercontent.com/1457576/115923534-a9690a00-a47e-11eb-9653-5c9e7bc1ff28.png">